cloudify_aws/eks/resources/node_group.py

```diff
@@ -87,6 +87,18 @@ def wait_for_nodegroup(self, params, status):
             }
         )
 
+    def start(self, params):
+        """
+            Updates the AWS EKS Node Group.
+        """
+        valid_keys = [
```

Comment:
Prefer to put this (line 94 - 99) in the operation (line 178), instead of here, which is just interface for calling API call.